clt_tests/tests/test-postgres-docker.rec

```diff
@@ -0,0 +1,37 @@
+––– block: ./base/init –––
+––– input –––
+docker network create app-network --driver bridge > /dev/null; echo $?
+––– output –––
+0
+––– input –––
+docker run -d --name postgres --network app-network -e POSTGRES_USER=postgres -e POSTGRES_PASSWORD=qazQASD! -e POSTGRES_DB=api_db -p 5432:5432 -v $(pwd)/clt_tests/tests/base/init.sql:/docker-entrypoint-initdb.d/init.sql postgres:latest > /dev/null 2>&1; echo $?
+––– output –––
+0
+––– input –––
+sleep 5; timeout 30 grep -m2 'database system is ready to accept connections' <(docker logs -f postgres 2>&1)
```

Comment:
Why not just timeout 35? Is there any difference?

Reply:
Fixed